README.adoc

```diff
@@ -59,20 +59,114 @@ corresponding Rudder version:
 
 ```
 - master (plugin compatible with Rudder next version, i.e developing branch)
-- branches/rudder/3.1 (plugins compatible with Rudder 3.1)
-- branches/rudder/4.1 (plugins compatible with Rudder 4.1)
+- branches/rudder/5.0 (plugins compatible with Rudder 5.0)
+- branches/rudder/5.1 (plugins compatible with Rudder 5.1)
 - etc
 ```
 
 This branch scheme allows to accomodate API changes between main Rudder versions.
 
 Most of the time, there is no need to recompile a plugin for the corresponding Rudder minor version, so that
-plugins compiled on branch `branches/rudder/4.1` should be compatible with all Rudder 4.1.x versions.
+plugins compiled on branch `branches/rudder/5.1` should be compatible with all Rudder 5.1.x versions.
 
 It may happens that at some point in the Rudder maintenance cycle, a Rudder minor version introduces a
 breaking change in a plugin API or a binary incompability in a plugin ABI. In such a case, we will
 explain which plugin versions are compatible with which Rudder versions in plugin readme file.
 
+=== Summary of binary compatibility for Rudder 5.0 branch
```

Comment:
This page is not part of Rudder doc but looks more like a developer doc, I think it should go elsewhere

Reply:
I'm not sure I understand the remark. Should the page be in Rudder doc?

Reply:
The general README from this rpeo in not is the doc (as it contains build instructions that not belong there, it should probable be a separate file we could directly include in the docs)

Reply:
OH, I can do that. Does a "version-compat.adoc" file suits your need? Do you have a better name for it?

Reply:
Sure!